run-tests.sh

```diff
@@ -23,8 +23,48 @@
 # waive the privileges and immunities granted to it by virtue of its status
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
-#isort -rc -c -df **/*.py && \
-#check-manifest --ignore ".travis-*" && \
-#sphinx-build -qnNW docs docs/_build/html && \
-python setup.py test
-#sphinx-build -qnNW -b doctest docs docs/_build/doctest
+# Quit on errors
+set -o errexit
+
+# Quit on unbound symbols
+set -o nounset
+
+check_script () {
+    shellcheck run-tests.sh
+}
+
+check_pydocstyle () {
+    pydocstyle cap_client
+}
+
+check_flake8 () {
+    flake8 .
+}
+
+check_manifest () {
+    check-manifest
+}
+
+check_pytest () {
```

Comment:
add 2 more functions for `check_pytest_unit` and `check_pytest_e2e` so we can run them separately

Reply:
added `check_pytest_unit`, what would be the command to run the `check_pytest_e2e`?